src/import/FSHImporter.ts

```diff
@@ -992,11 +996,30 @@ export class FSHImporter extends FSHVisitor {
   }
 
   visitAddElementRule(ctx: pc.AddElementRuleContext): AddElementRule {
-    const addElementRule = new AddElementRule(this.visitPath(ctx.path()))
+    const path = this.visitPath(ctx.path());
+    const addElementRule = new AddElementRule(path)
       .withLocation(this.extractStartStop(ctx))
       .withFile(this.currentFile);
 
     const card = this.parseCard(ctx.CARD().getText(), addElementRule);
+    if (isNil(card.min) || Number.isNaN(card.min)) {
```

Comment:
I see that lodash `isNil` checks if a value is null or undefined.  We typically just use double-equals (e.g., `card.min == null`) when we check this elsewhere in the codebase.  Is there a difference?

Reply:
Yeah, I think I'd prefer the consistency of using `==`.  Thanks!